browser/gui/app.cpp

```diff
@@ -123,6 +123,12 @@ App::App(std::string browser_title, std::string start_page_hint, bool load_start
         spdlog::critical("imgui-sfml initialization failed");
         std::abort();
     }
+    // ImGui tries to write to the current working directory. This can be problematic for a number of reasons,
```

Comment:
This comment doesn't really add anything. ImGui will try to write where we tell it to write, and you don't write why it's problematic. I'd just get rid of it.

Reply:
gone